lib/shopify_transporter/pipeline/magento/order/top_level_attributes.rb

```diff
@@ -46,7 +46,7 @@ def financial_status(hash)
             order_state = hash['state']
             status = nil
 
-            if order_state == 'Pending Payment'
+            if order_state == 'Pending Payment' || unpaid?(hash)
```

Comment:
Since 'pending' is the default state for orders in Shopify, should we make this the default on like 47?

Instead of `status = nil`, we can set `status = 'pending'`, and remove this if statement.

How do you feel about this?

Reply:
Good call! I'll update the code.